### PR TITLE
[Blog] Add Preprint Article on Service Mesh Performance Analysis

### LINKED ISSUE #388: [Blog] Post about "Exploring, Analyzing and Tuning Service Mesh Performance: A Literature Review" Paper
Create a blog post about my recent academic paper on Service Mesh Performance using Meshery and publish it at https://smp-spec.io/blog/

[Exploring, Analyzing and Tuning Service Mesh Performance: A Literature Review](https://www.techrxiv.org/articles/preprint/Exploring_Analyzing_and_Tuning_Service_Mesh_Performance_A_Literature_Review/22776119)

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
  - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by theBeginner86**
**Description**

This PR fixes #388

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: docs/collections/_posts/2023-08-26-exploring-analyzing-and-tuning-service-mesh-performance.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Each part of an app, called a “service” relies on other services to give users what they want. If a user of an online e-commerce store wants to by groceries, they need to know if the items they want, say “tomatoes” is in stock. So, the service that communicates with the e-commerce site’s inventory database needs to communicate with the product webpage, which itself needs to communicate with the users’s online shopping cart. To add business value, the owner of the site might eventually build a new service that gives users in-app product recommendation. Now this new service should also be cognizant of user’s shopping cart, and must be able to communicate with inventory database that the product page needed—it’s a lot of reusable, moving parts.
```
Comment (abhijeetgauravm) [inline docs/collections/_posts/2023-08-26-exploring-analyzing-and-tuning-service-mesh-performance.md:25]: @theBeginner86 there is a typo here it is 'by' . It should be 'buy'.

**Comment by gyohuangxin** [comment]
@theBeginner86 Can you update your PR according to @abhijeetgauravm 's comments?

**Review by leecalcote — APPROVED** [review]
Outside of the other feedback, LGTM

File: docs/collections/_posts/2023-08-26-exploring-analyzing-and-tuning-service-mesh-performance.md (hunk 1 above)
Comment (gyohuangxin) [inline docs/collections/_posts/2023-08-26-exploring-analyzing-and-tuning-service-mesh-performance.md:25]:
```suggestion
Each part of an app, called a “service” relies on other services to give users what they want. If a user of an online e-commerce store wants to buy groceries, they need to know if the items they want, say “tomatoes” is in stock. So, the service that communicates with the e-commerce site’s inventory database needs to communicate with the product webpage, which itself needs to communicate with the users’s online shopping cart. To add business value, the owner of the site might eventually build a new service that gives users in-app product recommendation. Now this new service should also be cognizant of user’s shopping cart, and must be able to communicate with inventory database that the product page needed—it’s a lot of reusable, moving parts.
```

**Review by gyohuangxin — APPROVED** [review]

**Pull request merged**